spec/requests/payment_methods_api_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe CompanyPaymentMethod do
```

Comment:
```suggestion
describe CompanyPaymentMethod do
```

Vocês podem tirar essa sintaxe hipster se quiserem 🤣

Reply:
obrigado kk